Automattic/wp-calypso
# Checklist Banner: Update greens/SVGs to match Muriel color standards

#### Changes proposed in this Pull Request

* Use `--color-success` for the gauge fill (forced with `!important`), with updated green success color inline as a fallback.
* Update SVGs for the check email graphic to use one of our dark blues.

**Before**

<img width="1266" alt="Screen Shot 2019-04-09 at 11 02 37 AM" src="https://user-images.githubusercontent.com/2124984/55811578-5a2e2780-5ab7-11e9-8dc3-44ed75476d58.png">

**After**

<img width="1280" alt="Screen Shot 2019-04-09 at 11 02 10 AM" src="https://user-images.githubusercontent.com/2124984/55811577-5a2e2780-5ab7-11e9-9486-95879ce30ef8.png">

#### Testing instructions

* Switch to this PR.
* Create a new account from `/start/` with a new email address (so you trigger the email validation notice)
* Go to your Stats page to see the Checklist Banner. 
* You should see the new colors in action!

Fixes #31649
